md/md_renderer.go

```diff
@@ -1,400 +1,400 @@
-	link := string(escape(node.Destination))
-	title := string(node.Title)
-	content := string(node.Literal)
-	r.outs(w, "[")
-	r.outs(w, content)
-	r.outs(w, "](")
-	r.outs(w, link)
-	if len(title) != 0 {
-		r.outs(w, ` "`)
-		r.outs(w, title)
-		r.outs(w, `"`)
-	}
-	r.outs(w, ")")
-}
-
-// RenderNode renders markdown node
-func (r *Renderer) RenderNode(w io.Writer, node ast.Node, entering bool) ast.WalkStatus {
-	switch node := node.(type) {
-	case *ast.Text:
-		r.text(w, node)
-	case *ast.Softbreak:
-		panic(fmt.Sprintf("node %T NYI", node))
-	case *ast.Hardbreak:
-		panic(fmt.Sprintf("node %T NYI", node))
-	case *ast.Emph:
-		r.emphasis(w, node)
-	case *ast.Strong:
-		r.strong(w, node)
-	case *ast.Del:
-		r.del(w, node)
-	case *ast.BlockQuote:
-		panic(fmt.Sprintf("node %T NYI", node))
-	case *ast.Aside:
-		panic(fmt.Sprintf("node %T NYI", node))
-	case *ast.Link:
-		r.link(w, node)
-	case *ast.CrossReference:
-		panic(fmt.Sprintf("node %T NYI", node))
-	case *ast.Citation:
-		panic(fmt.Sprintf("node %T NYI", node))
-	case *ast.Image:
-		r.image(w, node)
-	case *ast.Code:
-		r.code(w, node)
-	case *ast.CodeBlock:
-		r.codeBlock(w, node)
-	case *ast.Caption:
-		panic(fmt.Sprintf("node %T NYI", node))
-	case *ast.CaptionFigure:
-		panic(fmt.Sprintf("node %T NYI", node))
-	case *ast.Document:
-		// do nothing
-	case *ast.Paragraph:
-		r.para(w, node, entering)
-	case *ast.HTMLSpan:
-		r.htmlSpan(w, node)
-	case *ast.HTMLBlock:
-		r.htmlBlock(w, node)
-	case *ast.Heading:
-		panic(fmt.Sprintf("node %T NYI", node))
-	case *ast.HorizontalRule:
-		panic(fmt.Sprintf("node %T NYI", node))
-	case *ast.List:
-		r.list(w, node, entering)
-	case *ast.ListItem:
-		r.listItem(w, node, entering)
-	case *ast.Table:
-		panic(fmt.Sprintf("node %T NYI", node))
-	case *ast.TableCell:
-		panic(fmt.Sprintf("node %T NYI", node))
-	case *ast.TableHeader:
-		panic(fmt.Sprintf("node %T NYI", node))
-	case *ast.TableBody:
-		panic(fmt.Sprintf("node %T NYI", node))
-	case *ast.TableRow:
-		panic(fmt.Sprintf("node %T NYI", node))
-	case *ast.TableFooter:
-		panic(fmt.Sprintf("node %T NYI", node))
-	case *ast.Math:
-		panic(fmt.Sprintf("node %T NYI", node))
-	case *ast.MathBlock:
-		panic(fmt.Sprintf("node %T NYI", node))
-	case *ast.DocumentMatter:
-		panic(fmt.Sprintf("node %T NYI", node))
-	case *ast.Callout:
-		panic(fmt.Sprintf("node %T NYI", node))
-	case *ast.Index:
-		panic(fmt.Sprintf("node %T NYI", node))
-	case *ast.Subscript:
-		panic(fmt.Sprintf("node %T NYI", node))
-	case *ast.Superscript:
-		panic(fmt.Sprintf("node %T NYI", node))
-	case *ast.Footnotes:
-		// nothing by default; just output the list.
-	default:
-		panic(fmt.Sprintf("Unknown node %T", node))
-	}
-	return ast.GoToNext
-}
-
-// RenderHeader renders header
-func (r *Renderer) RenderHeader(w io.Writer, ast ast.Node) {
-	// do nothing
-}
-
-// RenderFooter renders footer
-func (r *Renderer) RenderFooter(w io.Writer, ast ast.Node) {
-	// do nothing
-}
+package md
```

Comment:
I have no idea why this entire file is being replaced. Not in UTF-8 or something?

Reply:
whitespace changes, CRLF (win) => LF (unix)